src/cluster/backend/zeromq/ZeroMQ.cc

```diff
@@ -73,6 +73,91 @@ constexpr DebugFlag operator&(uint8_t x, DebugFlag y) { return static_cast<Debug
 
 // NOLINTEND(cppcoreguidelines-macro-usage)
 
+ZeekProxyTelemetry::ZeekProxyTelemetry(zmq::socket_t&& arg_req) : req(std::move(arg_req)) {
+    // Registry telemetry metric callbacks with the manager. The callbacks run when someone
+    // scrapes the Prometheus endpoint.
+    zeek::telemetry_mgr->CounterInstance("zeek", "zeromq_frontend_msgs_received", {},
```

Comment:
I hadn't looked at the metric names. Would be good to add "cluster" and "proxy" and expand msgs to messages, but otherwise looks good:

```suggestion
    zeek::telemetry_mgr->CounterInstance("zeek", "cluster_zeromq_proxy_frontend_messages_received", {},
```